### PR TITLE
Update with new recipe for v2s1 manifest upload.

### PR DESCRIPTION
**Pull request by ipanova**
closes #3258
https://pulp.plan.io/issues/3258

### CODE REVIEW

File: docs/user-guide/recipes.rst [hunk 1]
```diff
@@ -356,56 +356,54 @@ both a JSON file for use with crane, and the static Image files to which crane
...
 
-.. note::
-
-    As of the time of this writing, ``skopeo copy`` can only output Docker v2
-    schema 2 content. Thus, only Docker v2 schema 2 content can be uploaded
-    to Pulp for now. In order to get your own Docker v2 schema 1 content into
-    Pulp, it is possible to run your own Docker registry and point Pulp's
-    feed URL at it and synchronize.
+Upload v2 schema 2 and schema 1 Images to Pulp
+---------------------------------------------
```
Comment (ammaritiz) [inline docs/user-guide/recipes.rst]: Title underline too short

File: docs/user-guide/recipes.rst [hunk 2]
```diff
@@ -440,6 +438,33 @@ The Blobs and Manifest are now in the Pulp repository::
     ``skopeo copy`` looses all the tags in the repository, therefore the manifests
     need to be tagged as a separate step after uploading it.
 
+  ::
+    $ pulp-admin docker repo search manifest --repo-id schema2
+
+    Created:      2018-02-14T16:06:12Z
+    Metadata:
+      Config Layer:       sha256:5b0d59026729b68570d99bc4f3f7c31a2e4f2a5736435641565
```
Comment (ammaritiz) [inline docs/user-guide/recipes.rst]: Unexpected indentation.

**Comment by ipanova** [comment]
@ammaritiz will you have other comments or i can merge this?

File: docs/user-guide/recipes.rst [hunk 3]
```diff
@@ -356,56 +356,54 @@ both a JSON file for use with crane, and the static Image files to which crane
...
-
-    As of the time of this writing, ``skopeo copy`` can only output Docker v2
-    schema 2 content. Thus, only Docker v2 schema 2 content can be uploaded
-    to Pulp for now. In order to get your own Docker v2 schema 1 content into
-    Pulp, it is possible to run your own Docker registry and point Pulp's
-    feed URL at it and synchronize.
+Upload v2 schema 2 and schema 1 Images to Pulp
+----------------------------------------------
 
 To upload a Docker Image to Pulp, first you must save its repository with Skopeo.
```
Comment (ammaritiz) [inline docs/user-guide/recipes.rst]: Maybe, add a hyperlink to [Skopeo](https://github.com/projectatomic/skopeo) here, otherwise, LGTM :+1:

**Pull request merged**